client/components/domains/register-domain-step/index.jsx

```diff
@@ -299,6 +301,16 @@ const RegisterDomainStep = React.createClass( {
 		} );
 	},
 
+	getSuggestionToken: function() {
+		const store = pick( SignupDependencyStore.get(), 'designType' );
```

Comment:
I'm not 100% sure if this is the best way to get this data. This should work just fine and is simple, but I had also considered whether it would be better to pass this information in as a prop.